examples/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,249 @@
+
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include <string.h>
+
+#include <FreeRTOS.h>
+#include <task.h>
+#include <ssid_config.h>
+
+#include <espressif/esp_sta.h>
+#include <espressif/esp_wifi.h>
+
+#include <paho/MQTTESP8266.h>
+#include <paho/MQTTClient.h>
+
+#include <semphr.h>
+
```

Comment:
I think the other files except for this have license headers. A LICENSE file for the eclipse public license may be a good idea, as that's new. 

Is this example something baoshi wrote, or is it original?